src/Analyzers/Helpers/RoslynExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DurableTask.Analyzers.Helpers;
```

Comment:
You don't _have_ to follow folder structure as your namespaces. If you want to put this in `Microsoft.DurableTask.Analyzers` that is fine. Then you don't need to add a `using` statement for it each time.

If you want that is - you can also leave it in the `Helpers` namespace if you prefer.

Reply:
I moved to `Microsoft.DurableTask.Analyzers`, much better than before - I really liked the flatten structure. Thanks!